### ISSUE TITLE
Does Macroquad work on FreeBSD?

### ISSUE THREAD

**Issue by kul-sudo** [not-fl3/macroquad#734]
When I run my project on Linux, a new windows opens up, and everything works correctly, but when I do the same on FreeBSD, the execution just immediately finishes, as if it's just an empty function. Could it be that I'm missing certain dependencies?

**Comment by not-fl3** [comment]
unfortunately I have zero experience with freebsd.
Feel free to post stacktraces and whatnot, maybe someone with freebsd knowledge will give some advice!

**Comment by kul-sudo** [comment]
> unfortunately I have zero experience with freebsd. Feel free to post stacktraces and whatnot, maybe someone with freebsd knowledge will give some advice!

Yeah, I use it isn't in the Supported Platforms, so that's why I'm wondering if there's a workaround, because this is a little bit frustrating.

**Comment by kul-sudo** [comment]
I actually think I could compile for Windows and use Wine.

**Comment by djtuBIG-MaliceX** [comment]
@kul-sudo If `ldd` doesn't reveal anything, can you see what happens with the executable running through `strace`?  Paste the stdout and it might reveal the reason why.

**Comment by kul-sudo** [comment]
> @kul-sudo If `ldd` doesn't reveal anything, can you see what happens with the executable running through `strace`?  Paste the stdout and it might reveal the reason why. 

Quite a good timing, because I'm going to install OpenBSD soon. When I get it on, I'll send all of that. I definitely remember that when I tried OpenBSD a year ago or so, I was unable to run Macroquad, so I should point out in the title that the issue exists on that OS too.
